ui/Screen/DesignSystemGuide.svelte

```diff
@@ -835,17 +909,20 @@
       <Swatch>
         <div style="display: flex;">
           <div style="position: relative; height: 200px; width: 280px;">
-            <TransactionAccordion
+            <TransactionCenter
+              summary={transaction.summarizeTransactions(transactions1)}
               transactions={transactions1}
               style="position: absolute; bottom: 0; right: 0;" />
           </div>
           <div style="position: relative; height: 200px; width: 280px;">
-            <TransactionAccordion
+            <TransactionCenter
+              summary={transaction.summarizeTransactions(transactions2)}
               transactions={transactions2}
               style="position: absolute; bottom: 0; right: 0;" />
           </div>
           <div style="position: relative; height: 200px; width: 280px;">
-            <TransactionAccordion
+            <TransactionCenter
+              summary={transaction.summarizeTransactions(transactions3)}
               transactions={transactions3}
               style="position: absolute; bottom: 0; right: 0;" />
           </div>
```

Comment:
The `TransactionStatusBar`'s are looking a little funny:

<img width="1021" alt="Screen Shot 2020-05-19 at 5 32 50 PM" src="https://user-images.githubusercontent.com/2712962/82391736-e7e88000-99f6-11ea-8ea4-580300278944.png">